inception-pdf-editor/src/main/java/de/tudarmstadt/ukp/inception/pdfeditor/PdfAnnotationEditor.java

```diff
@@ -74,4 +83,24 @@ private void handleError(String aMessage, Throwable aCause)
         error(aMessage + ExceptionUtils.getRootCauseMessage(aCause));
         return;
     }
+
+    /**
+     * Renders the annotation file for PdfAnno
+     * @param pdftxt Output string of PDFExtract
+     * @return Annotation file
+     */
+    public AnnoFile renderAnnoFile(String pdftxt) throws IOException
+    {
+        JCas jCas = getJCasProvider().get();
+        if (getModelObject().getProject() != null)
+        {
+            PdfExtractFile pdfExtractFile = new PdfExtractFile(pdftxt);
+            VDocument vdoc = render(jCas, 0, Integer.MAX_VALUE);
+            AnnoFile annoFile = PdfAnnoRenderer.render(getModelObject(),
+                vdoc, jCas.getDocumentText(), annotationService, pdfExtractFile);
+            return annoFile;
+        }
+
+        throw new IOException("");
```

Comment:
Provide a proper error message. Why is this an IOException anyway - there is no IO here? I'd probbaly simply do nothing if the project is null?